model-engine/model_engine_server/common/resource_limits.py

```diff
@@ -65,7 +65,7 @@ def validate_resource_requests(
     gpus: Optional[int],
     gpu_type: Optional[GpuType],
 ) -> None:
-    """Validates whether cpu/memory requests are reasonable"""
+    """Validates whether cpu/memory requests are reasonable. Shouldn't need to validate any nodes_per_worker in the multinode case"""
```

Comment:
one possible validation is nodes_per_worker must only be >1 if gpus > 8 or somethign like that. I don't imagine we want to support multinode for 1 or 2 gpus

Reply:
in theory yes, however I've been testing multinode on 2x1A10 nodes heh